### ISSUE TITLE
mAttacher.setScale works only after touching the screen

### ISSUE THREAD

**Issue by kndl22** [chrisbanes/PhotoView#403]
I'm using viewpager to display 4 different images using PhotoView.
Each fragment contains only 1 large image, and I want the fragment to initialize with scale so the user will see the image as it already scaled.

```
PhotoViewAttacher mAttacher = new PhotoViewAttacher(mapImg);
mAttacher.setMaximumScale(6);
mAttacher.setMediumScale(4);
mAttacher.setMinimumScale(2);
mAttacher.setScale(2,true);
```

The issue is that the scale won't happen untill i manually zoom-in the image with double tap. I tried adding GlobalLayoutListener so I ensure that the view is ready, but that works only if I don't remove the GlobalLayoutListener  and keep it alive -  this approach isn't a solution for me since there's alot of layout changes and the listener called each time causing a big mess.

What's the right approach to setScale to image immidiatly after loading it?

**Comment by smart-fun** [comment]
did you find a solution to this problem ?

**Comment by smart-fun** [comment]
You need to postpone the setScale to make it work:
```
PhotoViewAttacher mAttacher = new PhotoViewAttacher(mapImg);
mAttacher.setMaximumScale(6);
mAttacher.setMediumScale(4);
mAttacher.setMinimumScale(2);

mapImg.post(new Runnable() {
    @Override
    public void run() {
        mAttacher.setScale(2, true);
    }
});
```
